CefSharp/DevTools/DevToolsClient.cs

```diff
@@ -180,14 +252,16 @@ void IDevToolsMessageObserver.OnDevToolsEvent(IBrowser browser, string method, S
             //Only parse the data if we have an event handler
             if (evt != null)
             {
-                //TODO: Improve this
-                var memoryStream = new MemoryStream((int)parameters.Length);
-                parameters.CopyTo(memoryStream);
-
-                var paramsAsJsonString = Encoding.UTF8.GetString(memoryStream.ToArray());
+                var paramsAsJsonString = new StreamReader(parameters).ReadToEnd();
```

Comment:
What encoding is used here? Is it always UTF8 or is it locale specific? 

Does StreamReader implement Idisposable? If yes then we should wrap in using statement.

Reply:
As there are multiple calls we should extract out into a common method or add an extension method.